lib/main/localizations/app_localizations.dart

```diff
@@ -1933,4 +1933,15 @@ class AppLocalizations {
       name: 'ruleFilterAddressCcField',
     );
   }
+
+  String get youHaveCreatedANewFilter {
+    return Intl.message(
+        'You have created a new filter',
+        name: 'youHaveCreatedANewFilter');
+  }
+
+  String get youAreChangedYourFilterSuccessfully {
```

Comment:
`youHaveChangedYourFilterSuccessfully`

Reply:
Is your grammar correct? I don't trust much. =))

Reply:
ok.